packages/fast-components-react-base/src/checkbox/checkbox.tsx

```diff
@@ -166,8 +171,10 @@ class Checkbox extends Foundation<
     }
 
     private renderLabel(): React.ReactNode {
+        const { checkbox_label }: CheckboxClassNameContract = this.props.managedClasses;
+
         if (typeof this.props.label === "function") {
-            return this.props.label(get(this.props.managedClasses, "checkbox_label", ""));
+            return this.props.label(classNames(checkbox_label));
```

Comment:
nit: not necessary to declare the const checkbox_label, you could just do `classNames(this.props.managedClasses.checkbox_label)`

Reply:
it seems to be a matter of preference so I'm not blocking on it